### PR TITLE
Replacing the calls to ERFA by pure Julia functions for eraTttai

### PR DESCRIPTION
**Pull request by ashishpriyadarshiCIC**

### CODE REVIEW

**Comment by helgee** [comment]
First things first, since there is already a PR open for `eraTaitt` it is kind of rude to include it here. Please concentrate on `eraTttai`.

**Comment by ashishpriyadarshiCIC** [comment]
I apologise Sir, I will make the necessary changes

File: src/conversions.jl [hunk 1]
```diff
@@ -72,8 +73,16 @@ end
 # TAI <-> TT
 function rescale(::Type{TAIEpoch}, ep::TTEpoch)
     jd1, jd2 = julian1(ep), julian2(ep)
-    date, date1 = eraTttai(jd1, jd2)
-    TAIEpoch(date, date1)
+    dtat = OFFSET_TT_TAI/SECONDS_PER_DAY;
```
Comment (helgee) [inline src/conversions.jl]: The ported code should be in a separate function. Then you can also easily add a test for it.

**Comment by helgee** [comment]
Your PR has accumulated some conflicts that need to be resolved. The fact that you are working from master makes this more difficult than it should be. Please have another look at the recommended workflow (https://www.asmeurer.com/git-workflow/).

You also do not have to use pull requests to get the changes from the upstream (original) repo into your fork. Git can do all of these operations locally. If you have followed the instructions I linked above you could just use `git pull --rebase master` in your local branch to get the changes from upstream.

**Comment by helgee** [comment]
Please tell me if you need any assistance.

**Pull request closed without merging**